config/environments/production.rb

```diff
@@ -22,7 +22,7 @@
 
   # Disable serving static files from the `/public` folder by default since
   # Apache or NGINX already handles this.
-  config.serve_static_files = ENV['RAILS_SERVE_STATIC_FILES'].present?
+  config.serve_static_files = true
 
```

Comment:
This is the way I got heroku to serve static files - by setting it to true. But now that I'm doing local development I won't need it anymore. Thanks for pointing this out. Will revert back to original configuration